### PR TITLE
[Kernel] XBLA Fixes

### PR DESCRIPTION
**Pull request by randprint**
[kernel] 

Updated PR from Permanulled https://github.com/xenia-project/xenia/pull/1150/files#diff-358a33abb42660137d33300bbc8188c8R51

XGI messages to fix Jetpac XBLA

gets Aegis Wing, Jetpac, and MetalSlug 3 in game

### CODE REVIEW

**Comment by Margen67** [comment]
Two things:
Rebase and squash so it doesn't conflict and have unneeded commits
The commit should say [Kernel] not [kernel]

**Comment by illusion98** [comment]
Force pushed.

**Comment by randprint** [comment]
updated to correct behaviour of NetDll_WSARecvFrom() to bring State of Decay in game

File: src/xenia/kernel/xam/user_profile.cc [hunk 1]
```diff
@@ -18,8 +18,8 @@ namespace kernel {
 namespace xam {
 
 UserProfile::UserProfile() {
-  xuid_ = 0xBABEBABEBABEBABE;
-  name_ = "User";
+  xuid_ = 0xE000BABEBABEBABE;
+  name_ = "Xenia";
```
Comment (Razzile) [inline src/xenia/kernel/xam/user_profile.cc]: This should probably be moved to a separate issue/PR as it isn't related to this PR

Comment (randprint) [reply src/xenia/kernel/xam/user_profile.cc]: The Xuid change is most definitely related. The name, admittedly, is just cosmetic and could be left out from a merge. I'll probably leave it as-is until Rick addresses it.

**Pull request merged**